src/all.less

```diff
@@ -0,0 +1,4 @@
+@import "~esn-frontend-common-libs/src/all.less";
```

Comment:
We must check how this will work. As a library, the current code does not need to embed this common less file